CONTRIBUTING.md

```diff
@@ -237,6 +237,12 @@ $ git commit --amend
 $ git push --force-with-lease origin my-branch
 ```
 
+When the commits in your Pull Request get landed, they will be squashed into
+one commit, with metadata added to the commit message (including links to the
```

Comment:
OK, I think this commit https://github.com/nodejs/node/commit/0b5191f15d0f311c804d542b67e2e922d98834f8 could be a good example. There are a lot of lines changed, but it's still one logical change.